libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -33,12 +38,9 @@ public class MicrosoftAppCredentials : ServiceClientCredentials
         public const string MicrosoftAppPasswordKey = "MicrosoftAppPassword";
 
         /// <summary>
-        /// The token refresh code uses this client. Ideally, this would be passed in or set via a DI system to 
-        /// allow developer control over behavior / headers / timeouts and such. Unfortunately this is buried
-        /// pretty deep, the static solution used here is much cleaner. If this becomes an issue we could
-        /// consider circling back and exposing developer control over this HttpClient. 
+        /// The token refresh code uses this client. By sharing a MessageHandler here we can achieve the Static instance level
         /// </summary>
-        private static readonly HttpClient _httpClient = new HttpClient();
+        private readonly HttpClient _httpClient = new HttpClient();
```

Comment:
>rivate readonly HttpClient _httpClient = new HttpClient(); [](start = 9, length = 58)

I don't think this is right. When we switched to this static global singleton we went from 4rps to 150rps.  Are you 1000% sure that this will behave that way?

Reply:
Do you have the way to run those tests. I would like to do the same. I don't see a reason for performance degradation (since we are use the same delegating handlers). But still I would like to run the tests to be 1000% sure :D.